README.md

```
@@ -32,26 +32,47 @@ These trained models are integral components of **The SIM-ONE Framework**, servi
+**Modern Transformer Features**:
 - **RoPE (Rotary Position Embedding)**: Superior position encoding
-- **SwiGLU Activation**: ~10-15% performance improvement over standard activations
+- **SwiGLU Activation**: ~10-15% performance improvement
 - **RMSNorm**: Enhanced training stability
 - **Advanced BPE Tokenizer**: 32K vocabulary optimized for semantic preservation
 - **Governance Mechanisms**: Policy, memory, and trace attention heads
 - **Advanced Loss Functions**: Multi-objective optimization for content quality
 
-**Training Specifications:**
+**Training Configuration:**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)